src/auxiliary/math.jl

```diff
@@ -160,6 +160,18 @@ Given ε = 1.0e-4, we use the following algorithm.
     end
 end
 
+# TODO: Performance tuning for `ln_mean` (Float32), please find below
```

Comment:
```suggestion
# TODO: This may need some performance tuning, e.g., to optimize the
#       cutoff value `epsilon_f2` or the number of terms in the Taylor expansion
```